test/integration/component/test_ldap.py

```diff
@@ -21,13 +21,18 @@
 
 #!/usr/bin/env python
 from marvin.cloudstackTestCase import cloudstackTestCase
+#from marvin.cloudstackTestCase import *
 from marvin.cloudstackAPI import (updateConfiguration,
                                   createAccount,
                                   deleteAccount,
                                   addLdapConfiguration,
                                   deleteLdapConfiguration)
+
+
 from marvin.cloudstackAPI import login
-from marvin.lib.utils import cleanup_resources
+#from marvin.lib.utils import cleanup_resources
```

Comment:
Please remove code, do not comment it. It can be retrieved if necessary from Git.